tests/fe/fe_nedelec_sz_hanging_nodes_convergence.cc

```diff
@@ -0,0 +1,448 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 1998 - 2020 by the deal.II authors
```

Comment:
```suggestion
// Copyright (C) 2023 - 2024 by the deal.II authors
```
(Here and elsewhere.)

Reply:
The end year is not so important, because we have scripts doing it automatically. We try to be reasonably precise for the first year a file was created (which is why I made the comment), so feel free to make. But it is not super important, so either way is fine.